guides/doc-Provisioning_Guide/topics/Networking.adoc

```diff
@@ -225,7 +268,7 @@ ifeval::["{build}" == "satellite"]
 .For a NAT running on Red Hat Enterprise Linux 7:
 endif::[]
 ifeval::["{build}" == "foreman"]
-.For a NAT running on linux with firewalld service:
+.For a NAT running on Linux with `firewalld` service:
```

Comment:
NAT means Network Address Translation, so you cannot say "For a NAT".
You could say "Using NAT with `firewalld`

Reply:
@swadeley where would we be without your ever-watchful eye